src/main/io/osd.h

```diff
@@ -98,7 +98,8 @@ typedef enum {
 } osd_unit_e;
 
 typedef struct osdConfig_s {
-    uint16_t item_pos[OSD_ITEM_COUNT];
+    uint8_t item_pos[OSD_ITEM_COUNT][2];
+    bool visible[OSD_ITEM_COUNT];
```

Comment:
An array of

```
struct {
    uint8_t x;
    uint8_t y;
    bool visible;
}
```

would help keep the default positions tidier (should be more efficient to iterate over too).